misc/metrics/src/swarm.rs

```diff
@@ -123,6 +127,15 @@ impl Metrics {
             Box::new(connections_closed.clone()),
         );
 
+        let connection_establishment_duration = Family::new_with_constructor(
+            create_connection_establishment_duration_histogram as fn() -> Histogram,
+        );
+        sub_registry.register(
+            "connection_establishment_duration",
```

Comment:
```suggestion
            "connections_establishment_duration",
```

To be consistent with `connections_closed` and `connections_established`. Sorry for not catching this earlier.

Reply:
Can you please follow up here @John-LittleBearLabs. I think there is great value in consistent metric naming as that makes it easier for downstream users to discover them.